regression/pages/studio/course_outline_page.py

```diff
@@ -0,0 +1,72 @@
+"""
+Course Outline Page for Studio
+"""
+
+from edxapp_acceptance.pages.studio.overview import CourseOutlinePage
+
+from regression.pages.studio.utils import \
```

Comment:
nit: you can use parens instead of the slash. e.g., 
```python
from regression.pages.studio.utils import (
  click_confirmation_prompt_primary_button
)
```